Consistently use ICE candidate instead of ICE host candidate (#25)
youennf/mdns-ice-candidates: draft-ietf-rtcweb-mdns-ice-candidates.md

ICE Candidate Processing {#processing}
----------------------------

For any remote ICE candidate received by the ICE agent, the following procedure is used:

1. If the connection-address field value of the ICE candidate does not end with ".local" or if the value contains more than one ".", then process the candidate as defined in {{RFC8445}}.

2. Otherwise, resolve the candidate using mDNS.

> 3. If it resolves to an IP address, replace the value of the ICE candidate by the resolved IP address and continue processing of the candidate.

Reviewer: ```suggestion
3. If it resolves to an IP address, replace the mDNS hostname of the ICE candidate with the resolved IP address and continue processing of the candidate.
```

ICE Candidate Processing {#processing}
----------------------------

> For any remote ICE host candidate received by the ICE agent, the following procedure is used:

Reviewer: I think we should just drop 'host' entirely here. We won't worry about the type in this stage.

A peer-reflexive remote candidate could be learned and constructed from the
source transport address of the STUN Binding request as an ICE connectivity
check. The peer-reflexive candidate could share the same address as a remote

> ICE host candidate that will be signaled or has been signaled, received and is

Reviewer: This should probably say "remote mDNS candidate"

When there is no user consent, the following filtering should be done to prevent private IP address leakage:

> 1. ICE host candidates with an IP address are not exposed as ICE candidate events.

Reviewer: This section is obsoleted by #22
